python/ngen_config_gen/src/ngen/config_gen/models/cfe.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 
 from typing import Any, Dict, List, Union
```

Comment:
Since things like `Dict` and `List` were changed to `dict` and `list` are this imports required still?  This would be relevant across (potentially) all the files changed in this commit...

Reply:
Yeah, I didn't change the `pydantic` model type hints and I really should.
We will still have to use `typing.Optional` and `typing.Union` in `pydantic` model's for now, but we should be able to remove the others (I think). 

More plainly, _pydantic_ can't use `field: Foo | None = None` or `field: Foo | Bar` with `python<3.10`. See https://github.com/pydantic/pydantic/issues/2453#issuecomment-789110628